### DISCUSSION TITLE
Docmost as a One-Click App in CapRover

### DISCUSSION THREAD

**Discussion by larananas** [docmost/docmost#333] [Show and tell]
Hey everyone! 😊

I just made my first pull request to the CapRover One-Click Apps repo: [PR #1149](https://github.com/caprover/one-click-apps/pull/1149). It's my small contribution to add **Docmost as a One-Click App to CapRover**. For those who don't know, CapRover is this awesome opensource PaaS (Platform as a Service) that makes deploying apps super easy, and One-Click Apps are templates you can use to deploy common apps from a store with, well... just one click! 😎

Feel free to check it out and let me know if there are any issues—I’m open to feedback or improvements! 😄

Cheers!

**Comment by Philipinho** [comment]
Thank you for your contribution. I appreciate it.
